.github/workflows/acme-tests.yml

```diff
@@ -66,15 +66,15 @@ jobs:
             COPR_REPO=${{ needs.init.outputs.repo }}
           tags: pki-runner
           target: pki-runner
-          outputs: type=docker,dest=/tmp/pki-runner.tar
+          outputs: type=docker,dest=pki-acme-runner.tar
 
       - name: Upload runner image
```

Comment:
Minor nitpick - should we change the names of these tasks to be Cache rather than Upload?

Reply:
I think cache/upload is an implementation detail. Should we call these tasks Store & Retrieve instead?

Reply:
Sure, works for me.